Automattic/sensei
# Add Loading Indicator for Pattern List

Fixes #5166

### Changes proposed in this Pull Request

* Add a Loading overlay that appears only while the Pattern List is loading;

### Testing instructions

1. Access a WP installation with a plugin build containing the code saved on this branch;
2. Create a new course/lesson;
3. Advance to the last step of the modal (containing the pattern list);
4. Verify that a loading overlay appears while the pattern list is rendering;
5. Verify that the loading overlay disappears after the pattern list is mostly rendered;

### Screenshot / Video

#### Video of the progress overlay on the Courses modal

https://user-images.githubusercontent.com/529864/171471524-bce1f20b-fd90-4a27-a9a4-01fe6d88caae.mp4

#### Video of the progress overlay on the Lessons modal

https://user-images.githubusercontent.com/529864/171471545-0dbd2dfa-fdd2-4065-8f42-96544d622a79.mp4
